README.md

```diff
@@ -12,18 +12,18 @@
 
 ----
 
-The NuCypher network uses the [Umbral](https://github.com/nucypher/pyUmbral) 
-threshold proxy re-encryption scheme to provide cryptographic access control 
-for distributed apps and protocols. 
-Applications can use the NuCypher network to facilitate end-to-end encrypted 
-data sharing via sharing policies. Access permissions are baked into the 
-underlying encryption, and access can only be explicitly granted by the data owner. 
-Consequently, the data owner has ultimate control over access to their data. 
-At no point is the data decrypted nor can the underlying private keys be 
```

Comment:
Did you use an automated to to remove extra spaces? I'd like to use it too!

Reply:
Yes.  I wasn't sure and was considering turning it off because of those unintended changes but... if you approve :)